content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,234 @@
+the "Version info" and then click on the "Access model" button.
+
+Studio will present you with several ways of downloading models - with the CLI,
+in Python code and directly from your web browser. You can see all the web
+browser download steps here:
+
+<video width="99%" height="540" autoplay loop muted>
+    <source src="/img/mr-studio-download-model.webm" type="video/webm">
+</video>
+
+And here's how to do it with the CLI:
+
+First, configure the
+[DVC Studio Access Token](https://dvc.org/doc/studio/user-guide/account-and-billing#studio-access-token)
+(this only needs to be done once):
+
+```console
+dvc config --global studio.token <your Studio token>
+```
+
+Now you can use the following command to download the model:
+
+```bash
+dvc artifacts get https://github.com/<user>/example-get-started-experiments pool-segmentation
+```
+
+Here you just need to replace `<user>` with your GitHub user. This will download
+the latest version of the `pool-segmentation` model from the DVC Remote
+associated with the Git repository in the URL. You can also specify a different
+artifact version or a model registry stage. See the `dvc artifacts get`
+documentation for all options.
+
+If you don't have a Studio account at all, you can still use `dvc artifacts get`
+to download models, but you will need to provide the correct Git and DVC Remote
+credentials manually. You can see more details in the
+[documentation](/doc/command-reference/artifacts/get#description).
+
+## Connecting model registry actions to your CICD
+
+As we [noted](/docs/start/model-management/model-registry#GTO-tip) in the model
+registry chapter, all DVC model registry actions are captured in your Git
+repository as Git tags with a specific format.
+
+This also means that we can create CICD actions in our Git repository which will
+be triggered whenever versions are registered or stages are assigned.
+
+In the following, we will have a look at an example CICD workflow on GitHub
+which runs whenever we assign a version of our model to the "prod" stage in the
+model registry. The workflow simulates model deployment without the need to
+actually set up a deployment environment (so that you can test it easier) but it
+does include all the ingredients needed in an actual deployment job or any other
+CICD action.
+
+<admon type="tip">
+
+To see a real-world model deployment example you can check out a
+[similar workflow in our example repository](https://github.com/iterative/example-get-started-experiments/blob/main/.github/workflows/deploy-model-sagemaker.yml)
+which deploys a specific version of the model to an Amazon Sagemaker endpoint
+for inference whenever it is assigned to a stage.
+
+</admon>
+
+Go to the `.github/workflows/deploy-model-template.yml`. This is the file that
+GitHub uses to run our CICD workflow. You can see
+[runs of this workflow](https://github.com/iterative/example-get-started-experiments/actions/workflows/deploy-model-template.yml)
```

Comment:
Also need to check that this link works after merging the changes to the example repo.

Reply:
Captured in #4990